popup.css

```diff
@@ -0,0 +1,436 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+/* css sudo selector for entire css */
+
+/* stylelint-disable indentation */
```

Comment:
- [OPTIONAL ] Linters are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Please Instead of disabling the client, it will be nice to fix all linter errors manually.